### ISSUE TITLE
Automatic flow rate calibration with 0.2 mm nozzle fails!

### ISSUE THREAD

**Issue by Guenni1975** [bambulab/BambuStudio#3030]
### Bambu Studio Version

1.7.7.89

### Where is the application from?

Bambu Lab Official website

### OS version

Windows qq

### Printer

Bambu Lab X1c

### How to reproduce

1. go to Calibration 
2. select nozzle diameter 0.2 
3. select print bed type CoolPlate 
4. select all 4 filaments
5. start calibration.

### Actual results

The calibration starts and first carries out the bed levelling. Then the printer cleans the nozzle and loads the first filament. It then extrudes in the purge area and does not stop. Even cancelling the calibration does not help. It simply continues to extrude. Only disconnecting from the power supply, waiting briefly and switching on again interrupts the extrusion process.

### Expected results

It should calibrate the 4 filaments correctly.

### Project file & Debug log uploads

[noProjektFile.txt](https://github.com/bambulab/BambuStudio/files/13550092/noProjektFile.txt)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by Guenni1975** [comment]
Seems to work after all, just takes ages, hence my mistake, will be closed!

**Comment by dragon-eyes-bbl** [comment]
hi @Guenni1975 
Please submit a support ticket with machine log (not studio log) so the firmware team can investigate the problem cause its pause. 0.2mm is more challenging than other nozzles. 
Thanks

**Comment by davidschreiber** [comment]
Having the same issue as described here as soon as I try to perform flow rate calibration (or flow dynamics calibration for that matter) with the 0.2mm nozzle.

**Comment by mattkerrison** [comment]
Yep same problem, never had an issue on 0.4 nozzles, switched to 0.2 on my X1C and went to run calibration (Machine firmware 01.07.00.00, latest version of Bambu Studio on Windows) and it auto bed levels etc and then starts to extrude to clean the nozzle and just sits there when doing automatic flow rate calibration. Flow Dynamics worked but as others said, it just returns 0.6 K value for all PLA filaments so I don't think it's actually working

**Comment by ThePieMonster** [comment]
It feels crazy to be waiting 20+ mins just to get the initial purge done...

Most want to print at a 0.1 layer if they are using a 0.2 nozzle. 

You can also see the filament is bunching up since its purging for so long. This causes the purge to be even slower since there is now more resistance due to filament sitting in a blob below the nozzle. 

The firmware needs to clean the nozzle (perform a nozzle wipe) more often (and several times) for a single filament purge. 

<img width="302" alt="image" src="https://github.com/bambulab/BambuStudio/assets/11359134/2be96d68-385b-44a0-91c1-81813ebdf1ff">

**Comment by davidschreiber** [comment]
Either that, or do slight x axis movements while purging, so that there's no pileup of the filament and more distribution in the purge chute.

**Comment by mattkerrison** [comment]
Any update to this? Still seems to be happening in v1.91

**Comment by gaolegao-lx** [comment]
**Please upload log through Handy APP, and give us your ticket number**
***
***Upload log files through Bambu Handy***
     Log files from the Bambu printer can be uploaded via Bambu Handy over the network, with an upload speed of around 150 kbps under optimal network conditions.
     When uploading logs, you have the option to select a specific time frame, allowing you to reduce the volume of logs transmitted and shorten the upload duration.
     To upload files, go to **Me** → **Support Tickets** → Choose the ticket for log upload and proceed with the operation.
     ![app_support_ticket](https://wiki.bambulab.com/p1/troubleshooting/upload-log/app_support_ticket.png)
